sdk/python/v1beta1/setup.py

```diff
@@ -69,4 +69,7 @@
         "Topic :: Software Development :: Libraries :: Python Modules",
     ],
     install_requires=REQUIRES,
+    extras_require={
+		"huggingface": ["kubeflow-training[huggingface]"],
```

Comment:
shouldn't it be kubeflow-katib?

Reply:
I think it needs to be "kubeflow-training[huggingface]" because we need to reuse assets, such as the `storage-initializer`, from the Training SDK.

Reply:
Yes, that's right, it should be `kubeflow-training[huggingface]`